blockchain/chain_sync/src/bucket.rs

```diff
@@ -68,6 +80,36 @@ impl SyncBucketSet {
         // Return the heaviest tipset bucket
         vals.iter().max_by_key(|b| b.weight()).cloned()
     }
+    /// Updates SyncBucketSet by removing specified SyncBucket
+    pub(crate) fn _remove(&mut self, ts_bucket: &SyncBucket) {
```

Comment:
```suggestion
    pub(crate) fn remove(&mut self, ts_bucket: &SyncBucket) {
```

(same with below, I won't comment on each)